src/blocks/bulb-tf/controls.js

```diff
@@ -1,10 +1,5 @@
-import classnames from 'classnames';
-import icons from './icons';
-
-const { __ } = wp.i18n;
 const { Component } = wp.element;
 const { AlignmentToolbar, BlockControls, BlockAlignmentToolbar } = wp.editor;
-const { Toolbar, Button, Tooltip } = wp.components;
```

Comment:
No need to change this PR but for future purposes, these changes to TF could probably have been done on a separate pull request as they are not really related to topic of this PR ("the addition of the multiple choice question block").